proto/sample_v2_round_trip.json

```diff
@@ -3653,11 +3653,10 @@
             "warningAuditsGroupTitle": "Passed audits but with warnings", 
             "warningHeader": "Warnings: "
         }
-    },
+    }, 
     "lighthouseVersion": "4.3.0", 
     "requestedUrl": "http://localhost:10200/dobetterweb/dbw_tester.html", 
-    "runWarnings": [],
-    "stackPacks": [],
```

Comment:
This makes the proto round trip test fail b/c empty list is elided.  [] == nothing in proto as repeated.  This comes back to the whole ListValue thing.  See appveyor.

Reply:
Ha, yes I know. See my comment https://github.com/GoogleChrome/lighthouse/pull/8605#issuecomment-486534646 :)